### PR TITLE
Fixes #183 : Process.env object is not set properly

### LINKED ISSUE #183: [Bug]: Process.env object is not set properly
- CLIENT_IP & CLIENT_PORT is set to `undefined` in config.js because Process.env is not
  set  properly .

- Due to this `getWebAppStatus()` is not working properly which in turn shows the unreachable error in 
  the registration page .

### PR DESCRIPTION
**Pull request by vpn1997**
- Fixes #183

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "prefer-arrow-callback": 1,
     "no-useless-concat": 1,
     "prefer-template": 1,
-    "no-console": 1,
```
Comment (uttu357) [inline .eslintrc]: Why did you change this?

Comment (vpn1997) [reply .eslintrc]: I was using console methods that why i had to turn off no-console. I'll change it back to 1. :)
